services/spar/src/Spar/Sem/ScimExternalIdStore/Spec.hs

```diff
@@ -45,15 +46,17 @@ propsForInterpreter interpreter extract lower = do
     prop "insert/lookup" $ prop_insertLookup (Just $ show . void . extract) lower
     prop "insert/insert" $ prop_insertInsert (Just $ show . void . extract) lower
 
+-- FUTUREWORK: Add prop tests for missing operations
```

Comment:
have you given this a try?  if not i can do that.

Reply:
You can go ahead, but maybe better in a separate PR.